docs/sdkx-cocos2dx/notifications-android.mdx

```diff
@@ -0,0 +1,264 @@
+
+For example:
+
+```java
+import com.helpshift.HelpshiftCocosBridge;
+
+public class MyActivity extends Cocos2dxActivity {
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        // Add notificationSound config in install config map
+        Map<String, Object> config = new HashMap<>();
+        config.put("notificationSound", "<notification-sound-file-name>");
+        //...
+        // Install call
+        HelpshiftCocosBridge.install(
+            this,
+            "<App Id from the Helpshift Dashboard>",
+            "<Domain name from the Helpshift Dashboard>",
+            config);
+    }
+}
+```
+
+## Customizing notification channels {#notificationChannels}
+
+Starting from Android Oreo, Helpshift notifications will create a default channel named `In-app Support`. If you want to customize the name and description for the default channel, you can do so by using the `config` in the `install` call.
+
+Example:
+
+```java
+import com.helpshift.HelpshiftCocosBridge;
+
+public class MyActivity extends Cocos2dxActivity {
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        // Add notification Channel Id config in install config map
+        Map<String, Object> config = new HashMap<>();
+        config.put("notificationChannelId", "your channel name here");
+        //...
+        // Install call
+        HelpshiftCocosBridge.install(
+            this,
+            "<App Id from the Helpshift Dashboard>",
+            "<Domain name from the Helpshift Dashboard>",
+            config);
+    }
+}
+```
+
+## Configuring in-app notifications {#in-app}
+
+If you do not want the in-app notification support provided by the Helpshift SDK, you can set the flag to `false`. The default value of this flag is `true` i.e., in app notification will be enabled.
+
+- `Flag:` enableInAppNotification
+- `Values:` true/false
+- `Default:` true
+
+Example:
+
+```java
+import com.helpshift.HelpshiftCocosBridge;
+
+public class MyActivity extends Cocos2dxActivity {
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        // Add enableInAppNotification config in install config map
+        Map<String, Object> config = new HashMap<>();
+        config.put("enableInAppNotification", false);
+        //...
+        // Install call
+        HelpshiftCocosBridge.install(
+            this,
+            "<App Id from the Helpshift Dashboard>",
+            "<Domain name from the Helpshift Dashboard>",
+            config);
+    }
+}
+```
+
+## Showing notification count when replies are sent to the user {#showing-notification-count}
+
+To fetch unread messages count from the server you can call `requestUnreadMessageCount(boolean shouldFetchFromServer)` API. This API will return unread messages count via delegate.
```

Comment:
we should try to make it more clear that they need to register delegate listener for this.
Also make it clear in code that the impl shown is of delegate listener